### PR TITLE
feat: Remove Roles dropdown and align the sidenav to match latest designs & Fix sidenav items wrapping on hover

### PR DESCRIPTION
**Pull request by CharlRitterDev**
Remove Roles dropdown and align the sidenav to match latest designs & Fix sidenav items wrapping on hover

### Description
Remove Roles dropdown and align the sidenav to match latest designs & Fix sidenav items wrapping on hover

### Issue
https://spandigital.atlassian.net/browse/PRSDM-6329
https://spandigital.atlassian.net/browse/PRSDM-6206

### Checklist before merging

* [x] Did you test your changes locally?
* [x] Did you update the CHANGELOG?
* [ ] Is the documentation updated for this change? (If Required)

### CODE REVIEW

**Review by kelvinmanley — APPROVED** [review]

File: assets/_sass/bootstrap/bootstrap/_forms.scss
Comment (Zalaras) [inline assets/_sass/bootstrap/bootstrap/_forms.scss:1]: Is there a reason you removed stuff from bootstrap?

File: assets/_sass/bootstrap/bootstrap/_navbar.scss
Comment (Zalaras) [inline assets/_sass/bootstrap/bootstrap/_navbar.scss:1]: here too

**Review by Zalaras — APPROVED** [review]
LGTM, not tested locally. Note the bootstrap files you removed from, is that correct?

File: assets/_sass/bootstrap/bootstrap/_forms.scss
Comment (CharlRitterDev) [reply assets/_sass/bootstrap/bootstrap/_forms.scss:1]: The boostrap is a bit of a mess. Strictly speaking I don't need to remove it. I am just trying to slowly hack away at it when possible, as we'd want to remove the local bootstrap files completely.

The local copy is v3.3.7. We then pull in v4.2.1 in the baseof file, which is already a conflict issue. We also need to move to v5 for apple security.

So I could leave it in. I've removed the classname use in the HTML files. Just want to wack the weeds where possible.

**Pull request merged**